### PR TITLE
feat(ec2): add c5a instance class

### LINKED ISSUE #10781: [aws-ec2] Missing InstanceClass c5a
I can use c5a instance class on AWS Console and Cloudformation,
but I can't create c5a instance on CDK
because it is not declared in the InstanceClass enum

### Use Case

I want to use the instance
when creating container instance for my ECS cluster

### Proposed Solution

Add this line below the c5d declaration in InstanceClass enum
```typescript
    /**
     * Compute optimized instances based on AMD EPYC, 5th generation.
     */
    COMPUTE5_AMD = "m5a",
    /**
     * Compute optimized instances based on AMD EPYC, 5th generation
     */
    C5A = "c5a",
```

* [x] :wave: I may be able to implement this feature request
* [ ] :warning: This feature might incur a breaking change

---

This is a :rocket: Feature Request

### PR DESCRIPTION
**Pull request by xian13**
Closes #10781

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

**Comment by NetaNir** [comment]
Thanks!

File: packages/@aws-cdk/aws-ec2/lib/instance-types.ts [hunk 1]
```diff
@@ -193,6 +193,16 @@ export enum InstanceClass {
...
 
+  /**
+   * Compute optimized instances based on AMD EPYC, 5th generation.
+   */
+  COMPUTE5_AMD = 'c5a',
+
+  /**
+   * Compute optimized instances based on AMD EPYC, 5th generation
+   */
+  C5A = 'c5a',
```
Comment (NetaNir) [inline packages/@aws-cdk/aws-ec2/lib/instance-types.ts:204]: Why the AMD suffix? Can we keep the same convention as for c5?

```
  /**
   * Compute optimized instances, 5th generation
   */
  COMPUTE5 = 'c5',

  /**
   * Compute optimized instances, 5th generation
   */
  C5 = 'c5',
```

**Review by NetaNir — CHANGES_REQUESTED** [review]

File: packages/@aws-cdk/aws-ec2/lib/instance-types.ts (hunk 1 above)
Comment (xian13) [reply packages/@aws-cdk/aws-ec2/lib/instance-types.ts:204]: I use the same convention as m5a
```typescript
    /**
     * Standard instances based on AMD EPYC, 5th generation.
     */
    STANDARD5_AMD = "m5a",
    /**
     * Standard instances based on AMD EPYC, 5th generation
     */
    M5A = "m5a",
```

**Review by NetaNir — APPROVED** [review]

**Comment by NetaNir** [comment]
@mergifyio refresh

**Comment by NetaNir** [comment]
@Mergifyio update

**Review by NetaNir — APPROVED** [review]

**Pull request merged**